ui/app/helpers/format-utc.js

```diff
@@ -0,0 +1,11 @@
+import { helper } from '@ember/component/helper';
+import d3 from 'd3-time-format';
+
+export function formatUtc([date, specifier]) {
+  // given a UTC date, format and display it while preserving the UTC timezone.
+  const parsedDate = d3.isoParse(date);
```

Comment:
Why isoParse here instead of utcParse?

Reply:
Good catch! I originally was using `isoParse` to handle a variety of date formats (both strings and `Date` objects), but since we're currently only using strings in Zulu format I've replaced it with `utcParse` and added `isoParse` as a fallback. The benefit to `isoParse` is it will allow us to pass in non-UTC dates and the helper will convert and display those values as UTC.